_posts/2020-08-06-kubernetes-clients-comparison.md

```diff
@@ -0,0 +1,213 @@
+The charts and data seem to be very rudimentary, but improvements are expected to arrive over time.
+
+As most of the clients described in this post, Lens app is an open-source project.
+Mirantis is behind the development of Lens, but at the time of writing, no supported (paid) version is available.
+There is continuous active development on the app on the app and releases are about one month apart, so bugs fixes and new features should be available regularly.
+
+Screenshot: List of pods in Lens
+<img alt="Lens pod list" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-pod-list.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+Screenshot: Details about a pod in Lens, including Prometheus supplied metrics
+<img alt="Lens pod details" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-pod-details.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+Screenshot: Overview of a cluster in Lens, including the last error events
+<img alt="Lens cluster overview" src="{{ '/img/2020-08-06-kubernetes-clients-comparison/lens-cluster-overview.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 1000px;">
+
+## Kubenav
+[Kubenav](https://github.com/kubenav/kubenav){:target="_blank" rel="noopener noreferrer"} is a rich featured, open source Kubernetes client created in early 2020 to manage your cluster(s) with. 
+The application is under active development by the open-source community on their GitHub repository.
+There is cross-platform functionality, which means they provide you a desktop AND mobile client, which is a unique feature in the world of Kubernetes clients.
+They share the same codebase, so the navigation should be similar on all platforms.
+
+The navigation menu is self-explanatory as it categorizes every Kubernetes resource, which you can then filter by namespace in the top-right corner.
+They support all widely used Kubernetes resources and the status and configuration of those resources. 
+You can easily switch between Kubernetes clusters by using the drop-down menu available in the menu.
+
+There is a window that is available at all times where you can consult different deployment logs and a terminal. Each window is categorized under a tab.
+When you minimize this window, you will notice a blue shell icon in the right bottom corner of Kubenav, which allows you to open your current logs or terminal again.
+
+The app is made in the Ionic Framework using Capacitator as cross-platform runtime, which is something you notice when you start using the app.
+I don't really like this feeling, as I prefer my clients to have a more native feeling, although I understand why the developers chose this approach.
+
+The primary factor of why I would use this application is because they support almost every resource combined with a GUI if you don't like using the terminal.
```

Comment:
```suggestion
The primary reason why I would use this application is because they support almost every resource combined with a GUI if you don't like using the terminal.
```